src/query/delete.rs

```diff
@@ -1,3 +1,5 @@
+use std::rc::Rc;
+
```

Comment:
no longer needed
```suggestion
```

Reply:
Fixed in eb83cfe more-generic-into-iterator-instead-of-vec Remove useless precision from Vec turbofish